source/common/grpc/common.h

```diff
@@ -28,12 +28,47 @@ class Common {
   /**
    * Charge a success/failure stat to a cluster/service/method.
    * @param cluster supplies the target cluster.
+   * @param protocol supplies the downstream protocol in use, either gRPC or gRPC-Web.
```

Comment:
Per the other stats comments WRT naming, let's leave that for a different PR. We can definitely change things here if it makes sense but would like to think through that in an isolated change.